spec/models/article_spec.rb

```diff
@@ -469,17 +469,17 @@ def build_and_validate_article(*args)
     it "deindexes unpublished article" do
       expect do
         article.update(body_markdown: "---\ntitle: Title\npublished: false\ndescription:\ntags: one\n---\n\n")
-      end.to have_enqueued_job(Search::RemoveFromIndexJob).with(described_class.algolia_index_name, article.id).
-        and have_enqueued_job(Search::RemoveFromIndexJob).with("searchables_#{Rails.env}", article.index_id).
-        and have_enqueued_job(Search::RemoveFromIndexJob).with("ordered_articles_#{Rails.env}", article.index_id)
+      end.to have_enqueued_job(Search::RemoveFromIndexWorker).with(described_class.algolia_index_name, article.id).
```

Comment:
we don't have this helper, but as I mentioned you can adjust with these new helpers - see https://github.com/thepracticaldev/dev.to/issues/5305#issuecomment-570264567 - for example:

```ruby
sidekiq_assert_enqueued_with(job: Search::RemoveFromIndexWorker, args: [described_class.algolia_index_name, article.id], queue: "default") do
  article.update(body_markdown: "---\ntitle: Title\npublished: false\ndescription:\ntags: one\n---\n\n")
end
```